_documentation/user/flags/super-bullet.md

```diff
@@ -5,4 +5,14 @@ type: Good
 summary: Shoots through buildings. Can kill Phantom Zone.
 ---
 
-...
+The Super Bullet (SB) flag allows the player to fire shots that pass through objects, keeping their initial direction unchanged. An exception is such shots hitting teleporters, at which point they are transferred over them.
+
+Tanks holding the flag are also capable of killing Zoned opponents possessing the [Phantom Zone](../phantom-zone/) flag.
+
+## Tactics
+
+Super Bullet can be utilized as a means of defeating opponents that are on the map's same height level. It is potent on maps or in map sections with many objects, as the opponent is forced to navigate through said objects in order to target the holder, while the latter can directly target the former.
```

Comment:
I'd argue that the biggest pro for the SB flag is that it can shoot through objects to kill tanks that are possibly hidden inside drive-through objects or one-way walls. And that you can hide inside buildings and shoot tanks outside safely.